src/pages/BoardForm.tsx

```diff
@@ -1,3 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import { IcBack } from '../assets/icons';
+import BoardFormToast from '../components/BoardFormToast';
+import { FONT_STYLES } from '../styles/fonts/font';
+
 export default function BoardForm() {
-  return <div>BoardForm 화면</div>;
+  const navigate = useNavigate();
+
+  const [isActive, setIsActive] = useState<boolean>(false);
+  const [isToast, setIsToast] = useState<boolean>(false);
+  const [input, setInput] = useState<string>('');
+
+  useEffect(() => {
+    if (isToast) {
+      setTimeout(() => setIsToast(false), 1000);
+    }
+  }, [isToast]);
+
+  const changeInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.value.length > 5) {
+      setIsToast(true);
+    }
+    e.target.value.length > 0 ? setIsActive(true) : setIsActive(false);
+    setInput(e.target.value);
+  };
```

Comment:
```suggestion
  useEffect(() => {
    if (input.length > 5) {
      setInput((prev) => prev.slice(0, 5)); 
      setIsToast(true);
    }
  }, [input]);
  ...
   <input
        maxLength={5}
        type="text"
        value={input}
        onChange={(e) => setInput(e.target.value)}
      />
```
input에 maxLength를 주고 useEffect를 통해서 input의 길이가 변경될 때마다 체크해주는 건 어떨까요?

Reply:
코드 설명 까지 👍  저도 5글자 넘어가는 순간 **더이상 입력이 안되게 막고** 입력을 해도 **5글자 이상으로 늘어나지 않게** 하는 의견에 동의합니다!

Reply:
box-sizing을 border-box로 주고 padding:11px 10px;  이렇게 해봤는데, 너비를 넓히게 되면 어느 순간 이후에는 더 이상 늘어나지가 않아요  흠,, width를 100%로 주면 margin이 안먹고 어렵네여 증말
<img width="758" alt="스크린샷 2022-05-27 오후 12 05 38" src="https://user-images.githubusercontent.com/42725903/170621114-5c1253db-bfc0-4fd6-a256-414850494547.png">